deploy.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -eo pipefail
+set -x
+
+VAULT_TOKEN="$1"
+GIT_BRANCH="$2"
+DEPLOY_ENV="$3"
+
+set +x
+
+if [[ -z "${DEPLOY_ENV}" ]]; then
+    echo "DEPLOY_ENV argument not supplied; inferring from GIT_BRANCH '${GIT_BRANCH}'."
+
+    #TODO
+    if [[ "${GIT_BRANCH}" == "main" ]]; then
+        DEPLOY_ENV="dev"
+    elif [[ "${GIT_BRANCH}" == "alpha" ]]; then
+        DEPLOY_ENV="alpha"
+    elif [[ "${GIT_BRANCH}" == "perf" ]]; then
+        DEPLOY_ENV="perf"
+    elif [[ "${GIT_BRANCH}" == "staging" ]]; then
+        DEPLOY_ENV="staging"
+    else
+        echo "Git branch '${GIT_BRANCH}' is not configured to automatically deploy to an environment"
+        exit 1
+    fi
+fi
```

Comment:
I know this block exists in other deploy scripts … but is it useful? It implies we would maintain alpha, perf, and staging branches in github, and that's not our typical workflow

Reply:
true, I'll remove this and we can stick to tags